src/transport/msg_storage.py

```diff
@@ -1,40 +1,55 @@
-from typing import Any, Callable, Iterable, List
+from typing import Callable, Iterable, List
 
 from transport.msg_providers.common import BaseMessageProvider
+from transport.msg_types.common import BotMessage
 
 
 class MessageStorage:
-    messages: List = []
+    """
+    A storage class for managing and filtering messages fetched from various transports.
+
+    Attributes:
+        messages (List): A list storing all messages after filtering.
+    """
 
-    """Fetches all messages, filter them and storing"""
+    messages: List = []
 
     def __init__(self, transports: List[BaseMessageProvider], filters: List[Callable]):
         """
-        transports - List of objects with working get_messages method.
-        filters - functions that would be applied to messages when they are received. (That would need only one check)
+        Initializes the MessageStorage with a list of transports and filters.
+
+        Args:
+            transports (List[BaseMessageProvider]): A list of transport objects that provide a `get_messages` method
+                                                   for fetching messages.
+            filters (List[Callable]): A list of filter functions that take a message as input and return a boolean,
+                                      used to filter messages when they are received.
         """
         self._transports = transports
         self._filters = filters
 
-    def receive_messages(self) -> Iterable[dict]:
-        """Fetch all messages from transport and filter them"""
-        for transport in self._transports:
-            messages = transport.get_messages()
+    def get_messages_and_actualize(self, *args: Callable[[BotMessage], bool]) -> Iterable[BotMessage]:
```

Comment:
Better, not use *args. Let's just make it Iterable[Callable].
Or let's just receive one functions here (but with required filters inside) as in example in prev comment

Reply:
Replaced